src/Microsoft.ApiDesignGuidelines.Analyzers/Core/ImplementStandardExceptionConstructors.cs

```diff
@@ -110,24 +112,42 @@ private void AnalyzeSymbol(SymbolAnalysisContext context)
 
                 } //end of for loop
 
-                if (!defaultConstructorFound)
+                if (!defaultConstructorFound) //missing default constructor
```

Comment:
There's a bunch of repeated code in all these cases - could that be factored out?